### ISSUE TITLE
myrmex not doing anything [2.1.6-1.16.4]

### ISSUE THREAD

**Issue by ghost** [Alex-the-666/Ice_and_Fire#3797]
**Describe the bug**
none of the myrmex workers do anything after the initial 2 seconds of existence meaning that for the first 2 seconds you can trade and they will pick up resin but after that the only thing they want to do is attack players (if your rep is low enough) or they will find a single block that they all go to.

**To Reproduce**
1. Go to myrmex colony
2. (optional) spawn myrmex with spawn egg to see them work for 2 seconds

**Expected behavior**
i expected them to pick up resin and put it in a cocoon and do other myrmex things like trading.

**Comment by Mooonblaze** [comment]
Yes, same issue with the desert myrmex in the latest version. When their chunk was first loaded they worked, and over time the workers stopped coming out of the hive. Sentinels were positioned outside and attacked, and after they died, the workers and soldiers came out to attack. Once out of range the workers and soldiers went back into the hive, just a few blocks in, and sat there. They will not move unless a player gets close enough, then they will attack.

**Comment by darthvader45** [comment]
Agreed, this is still happening as of 2.1.11, and makes it impossible to get a hive up to 25 reputation to allow for neutrality and thus safe resin dropping, thus you can't reach 50 to make trading possible.  This means the queen egg is completely unobtainable.

**Comment by Mooonblaze** [comment]
Can confirm i’m still having myrmex be completely non-functional as of 2.1.11 as well. disappointed to see they haven’t been fixed in over a year

**Comment by darthvader45** [comment]
I've had some colonies go up to 5 or 10 rep, and one get to 20 (so damn close).  Wish I could've gotten em to 25 at least so I could safely drop resin nearby for them to collect.  That way I could get to 50 reputation and started trading with the queen for a queen egg.  Then I could have a colony of my own near my kingdom to not only give me cool stuff, but also serve as an autonomous defense for the kingdom.
